config/routes.rb

```diff
@@ -27,6 +27,11 @@
     end
   end
 
-  resources :issues, only: [:new, :create, :show]
+  resources :issues, only: [:new, :create, :show] do
+    resources :comments, module: :issues, only: [:create, :destroy]
```

Comment:
I don't understand you. Could you explain? If we remove :destroy from here, we won`t have this route, and if we remove whole block :only we will have more routes then we need.